package/yast2.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Fri Sep 16 17:53:15 UTC 2016 - kanderssen@suse.com
+
+- Network: Locate returns interface names instead of type.
```

Comment:
as martin always says, changelog note should describe the issue from the user's POV